packages/snaps-controllers/src/interface/SnapInterfaceController.ts

```diff
@@ -8,6 +8,7 @@ import type {
   ControllerStateChangeEvent,
 } from '@metamask/base-controller';
 import { BaseController } from '@metamask/base-controller';
+import type { InternalAccount } from '@metamask/keyring-internal-api';
```

Comment:
We already have the type for this in the repo: https://github.com/MetaMask/snaps/blob/503652fc34213986368b740a27cf699e3d82df42/packages/snaps-controllers/src/multichain/MultichainRouter.ts#L46-L57

Wondering if we should just re-use it?

Reply:
Agreed, I will move it to a shared place.